Formula/feedgnuplot.rb

```diff
@@ -16,7 +16,25 @@ class Feedgnuplot < Formula
 
   depends_on "gnuplot"
 
+  uses_from_macos "perl"
+
+  on_linux do
+    resource "List::MoreUtils" do
+      url "https://cpan.metacpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-0.430.tar.gz"
+      sha256 "63b1f7842cd42d9b538d1e34e0330de5ff1559e4c2737342506418276f646527"
+    end
+  end
+
   def install
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
+
+    resources.each do |r|
+      r.stage do
+        system "perl", "Makefile.PL", "INSTALL_BASE=#{libexec}"
+        system "make", "install"
+      end
+    end
+
     system "perl", "Makefile.PL", "prefix=#{prefix}"
     system "make"
     system "make", "install"
```

Comment:
Review is for after line 41 (i.e. need to be done after `prefix.install`). Linux test is still failing on `List::MoreUtils` since Perl scripts are not aware of local install.

Needs something like
```rb
    bin.env_script_all_files libexec/"bin", PERL5LIB: ENV["PERL5LIB"]
```

---

```
  ==> Testing feedgnuplot
  ==> /home/linuxbrew/.linuxbrew/Cellar/feedgnuplot/1.61/bin/feedgnuplot --terminal 'dumb 80,20' --exit
  Can't locate List/MoreUtils.pm in @INC (you may need to install the List::MoreUtils module) (@INC contains: /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/site_perl/5.34.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/site_perl/5.34.0 
```